universal-login-sdk/lib/api/deprecate.ts

```diff
@@ -0,0 +1,3 @@
+export function deprecateSDKMethod(subject: string) {
+  console.warn(`sdk.${subject} is deprecated. Use DeployedWallet.${subject} method instead.`);
+};
```

Comment:
Seems to me like `lib/api/` is not the right place for this little internal helper method. Maybe `core/utils`?

Reply:
Also, the name suggest that it is for deprecating ANY SDK method - but it is only for deprecating SDK methods which happen to be moved to `DeployedWallet`.

How about:
```
export function deprecateSDKMethod(subject: string, movedTo: string) {
  console.warn(`sdk.${subject} is deprecated. Use${movedTo}.${subject} method instead.`);
};
```
?